### PR TITLE
do not error if port missing

### PR DESCRIPTION
**Pull request by rphillips**

### CODE REVIEW

**Comment by goru97** [comment]
Do we have a test for this?

File: check/check_http.go [hunk 1]
```diff
@@ -87,19 +88,22 @@ func (ch *HTTPCheck) Run() (*ResultSet, error) {
 		return nil, err
 	}
 
-	host, port, err := net.SplitHostPort(parsed.Host)
-	if err != nil {
-		return nil, err
+	host, port, _ := net.SplitHostPort(parsed.Host)
+	if host == "" {
```
Comment (itzg) [inline check/check_http.go]: Should we also check for err here?

Comment (itzg) [inline check/check_http.go]: Nm, that's the point of the change :)

**Comment by goru97** [comment]
👍

**Comment by rphillips** [comment]
@itzg @goru97 tweaks to test for the default port.

**Comment by rphillips** [comment]
@itzg oh thanks!

**Pull request merged**